ops/lib.rs

```diff
@@ -300,6 +347,96 @@ fn codegen_v8_sync(
   }
 }
 
+fn can_be_fast_api(
+  core: &TokenStream2,
+  f: &syn::ItemFn,
+) -> Option<(TokenStream2, TokenStream2)> {
+  let inputs = &f.sig.inputs;
+  let output = &f.sig.output;
+
+  let ret = match is_fast_scalar(core, output) {
+    Some(ret) => ret,
+    None => return None,
+  };
+
+  // need a reciever for sequence types.
+  let mut needs_recv = false;
+  let mut args = vec![];
+  for input in inputs {
+    let ty = match input {
+      syn::FnArg::Typed(pat) => &pat.ty,
+      _ => unreachable!(),
+    };
+    match is_fast_scalar(core, ty) {
+      None => match is_fast_arg_sequence(core, ty) {
+        Some(arg) => {
+          needs_recv = true;
+          args.push(arg);
+        }
+        None => return None,
```

Comment:
Maybe add a comment about this being a "early return" out of the whole function since the type didn't match any Fast API arg type.